utils/index.js

```diff
@@ -47,6 +50,8 @@ export default {
 
   typeographyStyling: typeographyStyling,
 
+  randomQuote: randomQuote,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/dominion79/Saw-Front-End-App/pullRequest?prid=8928846)
```suggestion
  randomQuote,
```